src/app/window-handler.ts

```diff
@@ -987,21 +987,28 @@ export class WindowHandler {
       width: number;
     },
   ): void {
-    const parentWindow = BrowserWindow.getFocusedWindow();
     // Prevents creating multiple instances
-    if (didVerifyAndRestoreWindow(this.snippingToolWindow) || !parentWindow) {
+    if (didVerifyAndRestoreWindow(this.snippingToolWindow)) {
       logger.error('window-handler: Could not open snipping tool window');
       return;
     }
 
+    const electronWindows = BrowserWindow.getAllWindows();
+    const mainWindow = electronWindows[0];
```

Comment:
Will this work if BrowserWindow.getAllWindows() doesn't return any windows? Or maybe the assumption that there is always at least one window is fine